.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -9,8 +9,7 @@ assignees: ''
 ## Issue description
 
 <!--
-Before opening a bug, please take a look at the [Pact ReadtheDocs](https://pact-language.readthedocs.io/en/stable/).
-This explains Pact's semantics and some common issues and will also help you to find the information that the issue template asks for.
+Before opening a bug, please take a look at the [Pact documentation](https://docs.kadena.io/pact). This explains Pact's semantics and some common issues and will also help you to find the information that the issue template asks for.
```

Comment:
Alternately, perhaps you want to point to the language reference? Alas, the landing page for the language reference is quite bare.

```suggestion
Before opening a bug, please take a look at the [Pact documentation](https://docs.kadena.io/pact/reference). This explains Pact's semantics and some common issues and will also help you to find the information that the issue template asks for.
```

Reply:
I do favour the first proposal, as it contains also a general introduction.